bin/cheminfo-publish.js

```diff
@@ -177,11 +155,7 @@ You chose ${formatToBump(bump)} instead.`);
 
     // Add missing admins
     console.log('Adding missing admins');
-    for (var admin of adminList) {
-        if (owners.indexOf(admin) === -1) {
-            log(yield execNpm('owner add ' + admin));
-        }
-    }
+    log(yield execNpm(`access grant read-write ${org}:developers`));
```

Comment:
Is it possible to check first if the access is already granted? Then don't try to grant again.

Also is it possible to check if the current user is the package's first publisher ? Because only this person can execute this command.
If it is possible, I think we should print a message like "Please ask ${username} to run the following command locally: npm access grant read-write ${org}:developers ${packageName}"